src/calculateRentalCost.js

```diff
@@ -4,7 +4,14 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let price = days * 40;
+
+  if (days >= 7) {
+    price -= 50;
```

Comment:
Replace the hard-coded value `7` with a constant to explain what this value represents (e.g., `LONG_TERM_RENTAL`). Also, replace `50` with a constant (e.g., `LONG_TERM_DISCOUNT`).